oeo.omn

```diff
@@ -300,6 +300,9 @@ ObjectProperty: has_normal_state_of_matter
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "x has_normal_state_of_matter y if and only if a portion of x occurs in state y under normal pressure and temperature"
```

Comment:
```suggestion
<http://purl.obolibrary.org/obo/IAO_0000115> "x has_normal_state_of_matter y if and only if a portion of x occurs in state y under normal pressure (1 bar) and normal temperature (0°C)"
```

Reply:
There are several 'normal condition' depending on the application and I think it is better to clarify.